packages/stores/src/mockStore.ts

```diff
@@ -587,12 +589,34 @@ const mock = (): TStores & { is_mock: boolean } => {
         pushwoosh: {},
         contract_replay: {
             contract_store: {
+                accumulator_previous_spot_time: null,
+                barriers_array: [],
+                contract_config: {},
                 contract_info: {},
+                contract_update: {},
+                contract_update_history: [],
                 digits_info: {},
                 display_status: '',
+                getContractsArray: jest.fn(),
                 is_digit_contract: false,
                 is_ended: false,
+                marker: null,
+                markers_array: [],
             },
+            chart_state: '',
+            chartStateChange: jest.fn(),
+            has_error: false,
+            is_chart_loading: true,
```

Comment:
In _packages/core/src/Stores/contract-replay-store.js_ it has a default value `true`, but I'm still not sure as usually default values are `false`.